### ISSUE TITLE
Indentation for module test.yml changed after v2.5.1 release

### ISSUE THREAD

**Issue by drpatelh** [nf-core/tools#1866]
### Description of the bug

Running `nf-core modules create-test-yml` the indentation has been removed whenever a line begins with `-`. 

## Old format

```yaml
- name: samtools index test_samtools_index_bai
  command: nextflow run ./tests/modules/nf-core/samtools/index -entry test_samtools_index_bai -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/samtools/index/nextflow.config
  tags:
    - samtools
    - samtools/index
  files:
    - path: output/samtools/test.paired_end.sorted.bam.bai
      md5sum: 704c10dd1326482448ca3073fdebc2f4
```

## New format

```yaml
- name: samtools index test_samtools_index_bai
  command: nextflow run ./tests/modules/nf-core/samtools/index -entry test_samtools_index_bai -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/samtools/index/nextflow.config
  tags:
  - samtools
  - samtools/index
  files:
  - path: output/samtools/test.paired_end.sorted.bam.bai
    md5sum: 704c10dd1326482448ca3073fdebc2f4
```

**Comment by ewels** [comment]
All that should matter is that prettier is happier, right? As long as the syntax is the same? This should be forced via the `run_prettier_on_file` function that we have.

Then parsed syntax of these two examples is identical (try converting to JSON, it gives the same output).

Can you clarify what the problem is here @drpatelh ?

**Comment by ewels** [comment]
> This should be forced via the `run_prettier_on_file` function that we have.

@mashehu not sure that it does this actually. So maybe we can check that and add if not.

**Comment by mirpedrol** [comment]
Fixed by #2078 2078
